### PR TITLE
Re-adding packages

### PR DESCRIPTION
**Pull request by dnduffy**

### CODE REVIEW

File: src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config [hunk 1]
```diff
@@ -2,5 +2,8 @@
 <packages>
   <package id="log4net" version="2.0.5" targetFramework="net45" />
   <package id="Microsoft.ApplicationInsights" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
```
Comment (abaranch) [inline src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config]: Why do we need this in 4.5 project?..

**Comment by abaranch** [comment]
:shipit:

File: src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config (hunk 1 above)
Comment (dnduffy) [inline src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config]: I don't think I do but I was putting back all of the references which had existed in develop prior to the merges I made that took away some references that shouldn't have been removed.
I can try removing these again and see if the new build still passes.

Comment (abaranch) [inline src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config]: I do not see you putting back getlink. Is that OK?

Comment (dnduffy) [inline src/Adapters.Tests/Log4NetAppender.Net45.Tests/packages.config]: It was actually only in one Net40 project.

**Pull request merged**